src/guides/aaa_general/night-action-order.mdx

```diff
@@ -0,0 +1,34 @@
+---
+name: Basics
+menu: Night Action Order
```

Comment:
You'll want to add this menu item to the array in `doczrc.js` or it'll appear at the bottom of the list: https://github.com/werewolv-es/how-to-play/blob/3b7b12e3d741dff988cb63f45be589c761e27e9d/doczrc.js#L10

Reply:
Will do.

Reply:
Added